content/docs/2.13/scalers/cpu.md

```diff
@@ -16,7 +16,7 @@ go_file = "cpu_memory_scaler"
 KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include `requests` (at a minimum).
```

Comment:
nit: we may as well fix this sentence, at least that is my observation testing the webhook from https://github.com/kedacore/keda/pull/5377/
```suggestion
KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include at least one of `requests` or `limits`.
```

Reply:
My apologies @wozniakjan! I was busy with something. Thank you and @zroubalik for adding the commit in. :blush: